schema/pgsql/schema.sql

```diff
@@ -349,10 +349,11 @@ COMMENT ON INDEX idx_incident_history_time_type IS 'Incident History ordered by
 CREATE TABLE browser_session (
     php_session_id varchar(256) NOT NULL,
     username citext NOT NULL,
-    user_agent text NOT NULL,
-    authenticated_at bigint NOT NULL DEFAULT (EXTRACT(EPOCH FROM CURRENT_TIMESTAMP) * 1000),
+    user_agent varchar(4096) NOT NULL,
+    authenticated_at bigint NOT NULL,
 
-    CONSTRAINT pk_browser_session PRIMARY KEY (php_session_id, username, user_agent)
+    CONSTRAINT pk_browser_session PRIMARY KEY (php_session_id)
 );
 
-CREATE INDEX browser_session_authenticated_at_idx ON browser_session (authenticated_at);
+CREATE INDEX idx_browser_session_authenticated_at ON browser_session (authenticated_at DESC);
```

Comment:
For any sane database, specifying `DESC` should have no relevant effect in this case (but it doesn't hurt either, so feel free to keep it if you think it makes the purpose of the index easier to understand). Specifying a sort order in an index is mostly relevant if you have multiple columns where it changes the order in the index besides just reversing it entirely.

Reply:
This had more of a verbose meaning to it. We generally only want to read it in descending order as that's the single reason this index exists.